### ISSUE TITLE
[Bug] Firefox window not dragable due to hidden tabs preventing dragging

### ISSUE THREAD

**Issue by stitch** [piroor/treestyletab#3662]
## Abstract

Since a recent version of firefox tabs are non-draggable making it impossible to move the window with a lot of tabs. TST seems to hide all tabs when a window is focussed, so i cannot see what part of the window can be moved and what part cannot. Hope i'm in the right place with this report... sorry if i'm not.

## Steps to reproduce
![image](https://github.com/user-attachments/assets/33783975-b5bb-4ced-bcc2-7cd324c664aa)

## Used software
<img width="842" alt="image" src="https://github.com/user-attachments/assets/88995642-1034-4aaf-84ef-36e0c747a62f">

![image](https://github.com/user-attachments/assets/f94ed91d-f49a-418a-b013-ed1c8969ab41)

**Comment by irvinm** [comment]
@stitch I don't see how TST is related/responsible for this.  There is no way for TST to influence whether the native tabs are displayed or not.  I would imagine you might see this same behavior even without TST installed.  

Another thing you can try is to try to reproduce this in a new profile as a test with just TST installed.

**Comment by stitch** [comment]
@irvinm you are right. These things don't connect and i thought they did.

For anyone else experiencing problems: since a recent version of firefox the top bar does not support dragging the window anymore, instead it drags tabs. I was used to working exclusively with tree style tab and dragging the window with any cursor position on the top bar.

If you want the original tabs to disappear, use the below css in the procedure linked on this page:
https://superuser.com/questions/1424478/can-i-hide-native-tabs-at-the-top-of-firefox

```css
#tabbrowser-tabs tab { display: none !important; }
```

And then in Firefox using 'customize toolbar' to remove all other non-draggable items. This will leave you with a clean toolbar and just TST to do tab management.

![image](https://github.com/user-attachments/assets/653831a0-0ad6-4d21-80dd-0e566eb6bea9)

**Comment by stitch** [comment]
Then you will also want to use 'pin to toolbar' to click and see the sidebar again. Because if you press control +b by accident the bookmarks toolbar will show and your TST tab is gone. Clicking the icon will show the TST bar again.

![image](https://github.com/user-attachments/assets/b6ae901d-3e99-43b7-88b6-5643a2efc1e8)


You can also go to there using the dropdown at the top of the sidebar. But you might want to hide that too for decluttering.

![image](https://github.com/user-attachments/assets/2be4ffdf-ad38-4904-b79a-97feda977b62)
